playbooks/infrastructure/clevis.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Apply role clevis
+  hosts: "{{ hosts_clevis|default('clevis') }}"
+  serial: "{{ osism_serial['clevis']|default(osism_serial_default)|default(0) }}"
+
+  collections:
+    - osism.services
+    - community.crypto
```

Comment:
Why is this required here?

Reply:
@berendt  community.crypto is obselete i will fix it